### DISCUSSION TITLE
Despliegue de Statics JS

### DISCUSSION THREAD

**Discussion by jricardocam** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#417] [Q&A] [answered]
![1](https://user-images.githubusercontent.com/99063528/158502291-f8b76924-80fe-4271-a9cd-67f0e4291a7a.png)
![2](https://user-images.githubusercontent.com/99063528/158502296-5ccf4302-043b-431d-9b4b-28a1a3b439dd.png)
 Hola al ejecutar el codigo en el HTML se queda el ultimo valor y se le asigna a los dos.
¿Alguien sabe como corregirlo?

**Comment by MauroMontan** [comment, marked as answer]
es por que dentro de la función **estas asignando el mismo valor a las 2 variables diferente**. Puedes sacar las variables que contienen los elementos html de la función.
`
const Pokeesta = (url,elementoHtml) =>{
   elementoHtml.innerHTML = url;
}
`
de esta manera podras asignar un url diferente a cada elemento

**Reply by jricardocam** [reply]
Gracias Mauro
en el lugar de elementoHTML lo tengo que sustituir por ps o at?

**Reply by jricardocam** [reply]
en que parte tendria que poner el .getElementById?

**Reply by MauroMontan** [reply]
el codigo quedaría algo así, aunque realmente no necesitas una función para realizar esas asignaciones

`
const ps_element = document.getElementById("PS");
const at_element =  document.getElementById("att");

const Pokeesta = (url,elementoHtml) =>{ 
      elementoHtml.innerHTML = url; 

}

const ps = data.stats[0].base_stat;
const at = data.stats[1].base_stat;


Pokeesta(ps,ps_element);

Pokeesta(ps,at_element);


`

**Reply by jricardocam** [reply]
Mauro te agradezcon muchisimo!! ya funciona
soy nuevo en esto, a que te refieres con que no necesito la funcion para hacer las asignaciones?
